bentoml/deployment/utils.py

```diff
@@ -27,5 +27,5 @@
 
 def generate_bentoml_deployment_snapshot_path(service_name, platform):
     return os.path.join(
-        str(Path.home()), '.bentoml', 'deployment-snapshot', service_name, platform,
+        str(Path.home()), '.bentoml', 'deployment-snapshots', service_name, platform,
```

Comment:
discussed offline, also add service_version here

Reply:
done

Reply:
and change sequence to `'.bentoml', 'deployment-snapshots', platform, service_name, service_version, datetime`